generator/generator-opc-generator/generators/app/templates/dist/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title><%= elementName %></title>
+    <script src="<%= elementName %>.js"></script>
+    <style>
+        body {
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            height: 100vh;
+        }
+        <%= elementName %> {
+            --font-size: 40px;
```

Comment:
Why 40px?

Reply:
why --font-size

Reply:
Changed the size to 4 rem for a bigger view and changed the variable name to --op-component--text-size